rllab/dynamics_randomization/trpo_swimmer.py

```diff
@@ -0,0 +1,39 @@
+from rllab.algos import TRPO
+from rllab.baselines import LinearFeatureBaseline
+from rllab.envs.mujoco import SwimmerEnv
+from rllab.envs import normalize
+from rllab.policies import GaussianMLPPolicy
+from rllab.dynamics_randomization import RandomizedEnv
+from rllab.dynamics_randomization import Variations
+from rllab.dynamics_randomization import VariationMethod
```

Comment:
I propose we make this a wrapper which looks a lot like `normalized()`.

So we put these classes in `rllab.envs.mujoco.dynamics_randomization`. At the bottom of `dynamics_randomization.py` you can put
```python
# dynamics_randomization.py
randomize_dynamics = RandomizedDynamicsEnv
```

So that users can use it in a launcher like this
```python
# in your launcher
from rllab.envs.mujoco.dynamics_randomization import randomize_dynamics    # currently RandomizedEnv
from rllab.envs.mujoco.dynamics_randomization import Variations
from rllab.envs.mujoco.dynamics_randomization import Method    # currently VariationMethod
from rllab.envs.mujoco.dynamics_randomization import Distribution    # currently VariationDistribution

variations = Variations()
variations.randomize()\
          .at_xpath(".//geom[@name='torso']")\
          .attribute("density")\
          .with_method(Method.COEFFICIENT)\
          .sampled_from(Distribution.UNIFORM)\
          .with_range(0.5, 1.5)

env = randomize_dynamics(SwimmerEnv(), variations)

Reply:
Done

Reply:
Err, after trying the module I actually agree with some of your original naming. This module is general enough to randomize any part of an environment (good job!)

So the new vocabulary would be `randomize = RandomizedEnv`